launch_yaml/test/launch_yaml/test_reset.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test parsing a reset action."""
+
+import io
+import textwrap
+
+from launch.actions import ResetLaunchConfigurations, SetLaunchConfiguration
+from launch.frontend import Parser
+from launch.launch_context import LaunchContext
+
+
+def test_reset():
+    yaml_file = \
+        """\
+        launch:
+            - let:
+                name: 'foo'
+                value: 'FOO'
+            - let:
+                name: 'bar'
+                value: &bar 'BAR'
```

Comment:
@djchopp although PyYAML does support node anchors, I'd suggest we use launch substitution syntax i.e. `$(var ...)` to actually check things are working as expected. Same elsewhere.

Reply:
@hidimic Good call, fixed in [c31a37c](https://github.com/djchopp/launch/commit/c31a37cb95a453a0fb4f96967d4f67d5dbd76e90)